README.md

```diff
@@ -135,6 +139,38 @@ TARGET: LORA_E5
 
 https://www.seeedstudio.com/LoRa-E5-Wireless-Module-p-4745.html
 
+LoRa-E5 use only RFO_HP for TX power, so for all boards using this module you need to add this to your `mbed_app.json` on section `target_overrides`
+
+```json
+
+   "LORA_E5": {
+      "stm32wl-lora-driver.rf_switch_config": 2
+   }
+```
+
+## Seeed Studio LoRa E5 Mini
+
+Use LoRa E5 modulen basic board with USB/Serial and headers
+
```

Comment:
As you can see for other  targets, I added fora each one some information to match this doc file with the json file.
Which target should I build for this target ?

Thx
```
TARGET: XXX
```

Reply:
yep I updated readme with:

- `LORA_E5_DEV_BOARD`
- `LORA_E5_MINI`